setup.py

```diff
@@ -31,6 +31,7 @@ def read(fname):
                         'cloudpickle>=0.7.0'],
       tests_require=['pytest', 'flake8>=3.7.1', 'gitpython'],
       extras_require={'amici': ['amici>=0.10.21'],
-                      'petab': ['petab>=0.1.7']},
+                      'petab': ['petab>=0.1.7'],
+                      'hdf5': ['h5py>=2.10.0']},
```

Comment:
I would say we can have that as normal requirement, since it is the default way of storing stuff and not much overhead ...

Reply:
Wasn't sure / didn't check if that is the default. Then it *should* be required. Will change.

Reply:
not default for history, but for storing the result objects